commit

Meriel92 opened the pull request:
сделала пока что только под мобильную версию

Meriel92 committed:
commit

сделала только под мобильную версию

Oliko136 commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,188 @@
+  padding-top: 43px;
+  padding-bottom: 43px;
+
+  background: var(--secondary-text-color);
```
тут белый есть в переменных как --primary-bg-color, лучше тут использовать его как цвет фона из переменных, чтоб было понятней

Oliko136 commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,188 @@
+  height: 16px;
+
+  font-family: DM Sans;
+  font-style: normal;
```
font style normal можно не писать, это значение по умолчанию

Oliko136 commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,188 @@
+  width: 80px;
+  height: 16px;
+
+  font-family: DM Sans;
```
этот шрифт есть в переменных как var(--primary-font)

Oliko136 commented on src/partials/contacts.html:
```diff
@@ -15,13 +15,13 @@ <h3 class="contacts__title">Chicago</h3>
         <address class="contacts__adress">
-          <a href="tel:+61(0)383766284">+61(0) 383 766 284 </a>
-          <a href="malito:noreply@envato.com">noreply@envato.com</a>
+          <a class="contacts__adress--tel" href="tel:+61(0)383766284">+61(0) 383 766 284 </a>
```
очепятка - "address"

Oliko136 commented on src/partials/contacts.html:
```diff
@@ -15,13 +15,13 @@ <h3 class="contacts__title">Chicago</h3>
 
       <li class="contacts__item">
-        <p class="contacts-text-pre__title">FOOD TRUCK</p>
+        <p class="contacts-text-pre__title contacts__foodtrack">FOOD TRUCK</p>
```
очепятка - "foodtruck"

Oliko136 commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,188 @@
+  flex-direction: column;
+  margin-bottom: 54px;
+
+  @media screen and (min-width: 768px) and (max-width: 1279px) {
```
768px = переменная $tablet

Oliko136 commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,188 @@
+    margin-bottom: 81px;
+  }
+
+  @media screen and (min-width: 1280px) {
```
1280px = переменная $desktop

Oliko136 commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,188 @@
+  letter-spacing: 0.04em;
+
+  color: var(--secondary-text-color);
+  background: #f0d1a5;
```
background-color

Oliko136 commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,188 @@
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
```
тут line height через множитель забыла сделать) и удалить коммент из фигмы внизу

Oliko136 commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,188 @@
+
+.contacts__line {
+  margin-bottom: 25px;
+  margin-top: 25px;
```
маржин топ не нужно, так как ты уже писала маржин ботом верхнему элементу, он тогда будет в два раза больше

Meriel92 replied:
не получается без маржина топа, слишком близко получается текст к тексту, а с топом визуально все ок, я пока оставила, посмотри

Oliko136 commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,188 @@
+}
+
+.contacts__card {
+  display: flex;
```
в мобильной версии должно работать без флексов, так как все и так идет в колонку. Можно задать тут просто маржин 0 авто

Oliko136 commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,188 @@
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
```
тут и в десктопной flex-wrap и flex-direction не нужен: flex-wrap потому что это используется для переноса на новую строку, а тут только одна строка; а flex-direction потому что если ты удалишь флекс в колонку в моб версии, то тут флекс будет по умолчанию в строку

Meriel92 committed:
Feature/hero (sass/layout)

Meriel92 committed:
Update _contacts.scss

из contacts-card  удалила  justify content center и align items center и заменила на  (max-width: 767) margin-right: auto; margin-left: auto
